### DISCUSSION TITLE
post/android-aosp/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#18] [Announcements]
# post/android-aosp/

前言前段时间主力自用的 LG G8 (alphalm) 出了一些小毛病, 且因为驱动闭源没有完美的类原生, 用起来不太顺手就闲鱼出了. 后来拿到了家长退役的一台 Redmi Note 10 pro (chopin) 做主力机. 个人还是比较依赖谷歌生态和习惯使用原生系统的, 所以自然是要给它刷机啦0w0. Note 10 pro 机型相关虽然本来想写的是我使用类原生做的一些调教, 但还是先跑题扯一

https://blog.lyc8503.site/post/android-aosp/

**Comment by hmd3922** [comment]
老哥，我也刷了gsi，但是不知道怎么优化，现在无法打电话收发短信，提示不可用，用的电信

**Reply by lyc8503** [reply]
可能是 VoLTE 问题，可以尝试调整下 IMS，我移动/联通卡都没遇到过这个问题

**Reply by hmd3922** [reply]
谢啦老哥，看见phh这个东西了，还在调整

**Reply by hmd3922** [reply]
这手机跟别的不一样的就是解avb的时候多两个步骤，之前就栽坑里了，无限重启了一整天

**Comment by awkoo** [comment]
cdroid好像是从lineageos改的，不如直接用lineageos
